tests/integration/test_definitions.py

```diff
@@ -45,18 +54,80 @@ def define_tests():
             "local",
         ),
     }
-    server_tests = {
-        "merlin server init": ("merlin server init", HasRegex(".*successful"), "local"),
+    server_basic_tests = {
+        "merlin server init": (
+            "merlin server init",
+            HasRegex(".*successful"),
+            "local",
+            "rm -rf appendonly.aof dump.rdb merlin_server/",
+        ),
         "merlin server start/stop": (
-            "merlin server start; merlin server status; merlin server stop",
+            """merlin server init;
```

Comment:
Pardon my ignorance of some of the larger machinery in here, but what do the ';'s add here that isn't already accomplished by the new lines embedded in those multi-line strings (I'm assuming this is a splitting mechanism to enable accessing the individual commands?)

Reply:
The addition of ';'s is because the command are all run with a shell. The testing framework doesn't parse the commands but runs the string with subprocess.Popen() with shell. With most shell-scripts ";" is to designate the end of a command. While spaces and new-lines are used for separating arguments of a given command.